src/commands/registries/azure/tasks/scheduleRunRequest.ts

```diff
@@ -62,7 +75,7 @@ export async function scheduleRunRequest(context: IActionContext, requestType: '
                 isPushEnabled: true,
                 sourceLocation: uploadedSourceLocation,
                 platform: { os: osType },
-                dockerFilePath: fileItem.relativeFilePath
+                dockerFilePath: path.relative(rootUri.fsPath, fileItem.absoluteFilePath)
             };
         } else {
             runRequest = {
```

Comment:
Can you change `taskFilePath` in this one to match the `path.relative(...)` above used for the Dockerfile path?

Reply:
Done